lua/starfall/sflib.lua

```diff
@@ -478,11 +478,15 @@ end
 
 if SERVER then
 	util.AddNetworkString( "starfall_requpload" )
+	util.AddNetworkString( "starfall_uploadlist")
```

Comment:
Be careful of codestyle here. Few other spots in this file which slipped through. I would have made a suggestion, but GitHub review lacks the ability to do multiple line suggestions.